dev/client.scss

```
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+html,
+body {
+    padding: 0;
+    margin: 0;
+}
+
+#app {
+    display: flex;
+    flex-direction: column;
```

Just a suggestion but you could simplify this and it would allow you to remove the entire media query and flexbox styling
```suggestion
    display: grid;
    grid-template-columns: repeat(2, minmax(384px, 1fr));
```

---

🤔 Grid columns won't wrap by default, or am I missing something?

---

Yes it will since we are explicit with the column count. Here we are saying there will be 2 columns and the minimum size of that column is 384px (768/2) and the max is set 1 fractional unit so each column will be equal. If the column size goes below 384px it will automatically drop it down to the next row and fill that space.

---

It definitely doesn't seem to drop down automatically: https://codepen.io/MattCowley/pen/gOvqPem

---

Ah, looks like it needs `repeat(auto-fit, ...`?: https://codepen.io/MattCowley/pen/xxYMZQx

---

My bad, I missed adding a keyword. Try `grid-template-columns: repeat(auto-fill, minmax(384px, 1fr));` 

Another option could be to do it like this since technically on mobile those are just block elements.
```
@media only screen and (min-width: 768px) {
  #app {
    display: grid;
    grid-template-columns: repeat(2, 1fr);
  }
}
```